frontend/packages/operator-lifecycle-manager/integration-tests-cypress/tests/createNamespace.ts

```diff
@@ -0,0 +1,55 @@
+import { checkErrors, testName } from '../../../integration-tests-cypress/support';
+import { listPage } from '../../../integration-tests-cypress/views/list-page';
+import { modal } from '../../../integration-tests-cypress/views/modal';
+
+describe('Create namespace from install operators', () => {
+  before(() => {
+    cy.login();
+    cy.createProject(testName);
+  });
+
+  afterEach(() => {
+    checkErrors();
+  });
+
+  after(() => {
+    cy.deleteProject(testName);
+    cy.logout();
+  });
+
+  const nsName = `${testName}-ns`;
+
+  it('creates namespace from operator install page', () => {
+    const operatorSelector = '3scale-community-operator-community-operators-openshift-marketplace';
+    cy.log('test namespace creation from dropdown');
+    cy.visit(`/operatorhub/ns/${testName}`);
+    cy.byTestID(operatorSelector).click();
+    cy.byLegacyTestID('operator-install-btn').click({ force: true });
+
+    cy.get('#dropdown-selectbox').click();
```

Comment:
Hi, we are trying to stay away from using css id (#) selectors and using [Cypress best practices](https://github.com/openshift/console/blob/master/frontend/packages/integration-tests-cypress/README.md#best-practices) for selecting elements.   
A notable exception is if the elements is a third-party dependency where we cannot add a `data-test` attribute.

Reply:
Added `data-test` attribute as suggested